spec/lib/api/1.1/skus-spec.js

```diff
@@ -16,7 +17,9 @@ describe('Http.Api.Skus', function () {
         ]);
         return helper.startServer([
             dihelper.simpleWrapper(workflowApiService, 'Http.Services.Api.Workflows')
-        ]);
+        ]).then(function() {
+            _ = helper.injector.get('_');
```

Comment:
`_` has already in the global environment, you don't need to do this. see this line https://github.com/RackHD/on-core/blob/master/spec/helper.js#L15